src/tool_operate.c

```diff
@@ -195,6 +196,18 @@ static int sockopt_callback(void *clientp, curl_socket_t curlfd,
             tos, error, strerror(error));
     }
   }
+#endif
+#ifdef SO_PRIORITY
+  if(config->vlan_priority > 0) {
+    int priority = (int)config->vlan_priority;
+    if(setsockopt(curlfd, SOL_SOCKET, SO_PRIORITY,
+      (const char *)&priority, sizeof(priority)) != 0) {
+      int error = errno;
+      warnf(config->global, "SO_PRIORITY %d failed with errno %d: %s;\n",
```

Comment:
The user will not know what `SO_PRIORITY` is. They used `--vlan-priority`.

Reply:
Reworded.